libr/magic/mdump.c

```diff
@@ -210,7 +211,7 @@ const char *file_fmttime(ut32 v, int local, char *pp) {
 		}
 #endif /* HAVE_TM_ISDST */
 #endif /* HAVE_DAYLIGHT */
-		if (daylight) {
+		if (&daylight) {
```

Comment:
```suggestion
		if (*daylight) {
```

you need to access the contents with *, not the address of the reference